chromium_src/components/content_settings/core/browser/content_settings_registry.cc

```diff
@@ -17,7 +17,7 @@ void ContentSettingsRegistry::BraveInit() {
            CONTENT_SETTING_BLOCK, WebsiteSettingsInfo::SYNCABLE,
            AllowlistedSchemes(),
            ValidSettings(CONTENT_SETTING_ALLOW, CONTENT_SETTING_BLOCK),
-           WebsiteSettingsInfo::SINGLE_ORIGIN_ONLY_SCOPE,
+           WebsiteSettingsInfo::TOP_ORIGIN_ONLY_SCOPE,
```

Comment:
What's the difference between `TOP_ORIGIN_ONLY_SCOPE` and `REQUESTING_ORIGIN_ONLY_SCOPE` and `GENERIC_SINGLE_ORIGIN_SCOPE`?

Reply:
* `REQUESTING_ORIGIN_WITH_TOP_ORIGIN_EXCEPTIONS` - the requesting origin is used, and it's either checked with no specified top-origin, or with a specified top-origin
 * `TOP_ORIGIN_WITH_RESOURCE_EXCEPTIONS_SCOPE` - the top-origin is used, and resource-url exceptions can be added
 * `TOP_ORIGIN_ONLY` / `REQUESTING_ORIGIN_ONLY` - only the top-level origin / requesting origin is used
 * `GENERIC_SINGLE_ORIGIN` - scoped to a single origin but does not fit the paradigm of top/requesting origins (usually for website settings)
 * `REQUESTING_AND_TOP_ORIGIN` - both origins need to be always specified

I was hoping that during review someone who has better understanding of this area could chime in and determine if these replacements are correct, or require any adjustments.

Reply:
`TOP_ORIGIN_ONLY_SCOPE` is correct. REQUESTING_ORIGIN would be the frame, but we apply shields settings based on the main frame